docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -25,6 +25,22 @@ Running {agent} standalone? Also refer to <<debug-standalone-agents>>.
 //TODO: Improve how content is organized in this section, perhaps by breaking
 //it down into sections like Agent, Fleet Server, etc.
 
+[discrete]
```

Comment:
This section has already been reviewed in https://github.com/elastic/observability-docs/pull/1780.